src/elfpy/strategies/single_lp.py

```diff
@@ -10,29 +10,14 @@
 
 
 class Policy(BasicPolicy):
-    """
-    simple LP
-    only has one LP open at a time
-    """
-
-    def __init__(self, market, rng, wallet_address, budget=1000):
-        """call basic policy init then add custom stuff"""
-        self.amount_to_lp = 100
-        super().__init__(
-            market=market,
-            rng=rng,
-            wallet_address=wallet_address,
-            budget=budget,
-        )
+    "single LP: only has one LP open at a time"
+    amount_to_lp = 100
 
     def action(self):
-        """
-        implement user strategy
-        LP if you can, but only do it once
-        """
-        action_list = []
         has_lp = self.wallet.lp_in_wallet > 0
         can_lp = self.wallet.base_in_wallet >= self.amount_to_lp
         if can_lp and not has_lp:
-            action_list.append(self.create_agent_action(action_type="add_liquidity", trade_amount=self.amount_to_lp))
-        return action_list
+            self.action_list.update(
```

Comment:
It appears as though you deleted the docstring. Please put it back. Also, I think the action_list should not be a member variable of agent (mentioned in an earlier comment).

Reply:
added docstrings everywhere